app/static/messages.properties

```diff
@@ -74,6 +74,7 @@
     "openstack_credential_success": "Openstack credential '{0}' created successfully",
     "openstack_credential_oendpoint_invalid": "invalid Openstack endpoint min 5 char",
     "openstack_credential_otenantName_invalid": "invalid Openstack tenantname min 5 char",
+    "openstack_credential_odomainName_invalid": "invalid Openstack domainname min 5 char",
```

Comment:
I would use 'domain name' and 'tenant name' one line above in the error messages.